### PR TITLE
Feature/15007 15097 automation

### PR DESCRIPTION
**Pull request by sinhaamit1980**

### CODE REVIEW

File: admin/test/features/edit_check_window.feature [hunk 1]
```diff
@@ -0,0 +1,22 @@
+
+Feature: Edit Check Window
+
+
+  Background:
+    Given I am logged in with a test developer
+    And I am on the manage check windows page
+    And I want to edit a previously added check
```
Comment (activemq) [inline admin/test/features/edit_check_window.feature]: Can we summarize these 3 steps in to one  ? 'Given I want to edit a previously added check' maybe?

File: admin/test/features/step_definitions/edit_check_window_steps.rb [hunk 2]
```diff
@@ -0,0 +1,70 @@
+
+And(/^I want to edit a previously added check$/) do
+  manage_check_window_page.create_new_window.click
+  @page = add_edit_check_window_page
+  step 'I create new check window with valid data'
+  @chk_window = manage_check_window_page.find_check_row(@check_window_name)
+  @chk_window.check_window.click
+  # @page = add_edit_check_window_page
```
Comment (activemq) [inline admin/test/features/step_definitions/edit_check_window_steps.rb]: Remove any comments please

File: admin/test/features/support/page_objects/manage_check_window_page.rb [hunk 3]
```diff
@@ -1,20 +1,29 @@
 class ManageCheckWindowPage < SitePrism::Page
-  set_url '/choose-check-window'
+  set_url '/administrator/check-windows'
 
   element :heading, '.heading-xlarge', text: 'Manage check windows'
   element :page_instructions, '.lede', text: 'Create, edit or remove check windows. Test development will be responsible for assigning check forms to the check windows created here.'
-  element :create_new_window, 'input[value="Create new check window"]'
+  # element :create_new_window, 'input[value="Create new check window"]'
```
Comment (activemq) [inline admin/test/features/support/page_objects/manage_check_window_page.rb]: Remove any comments please

Comment (sinhaamit1980) [reply admin/test/features/support/page_objects/manage_check_window_page.rb]: Done

File: admin/test/features/step_definitions/edit_check_window_steps.rb (hunk 2 above)
Comment (sinhaamit1980) [reply admin/test/features/step_definitions/edit_check_window_steps.rb]: Done

File: admin/test/features/edit_check_window.feature (hunk 1 above)
Comment (sinhaamit1980) [reply admin/test/features/edit_check_window.feature]: Done

**Pull request merged**